Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -200,72 +198,45 @@ extension ManifestLoaderProtocol {
         dependencyMapper: DependencyMapper,
         fileSystem: FileSystem,
         observabilityScope: ObservabilityScope,
-        delegateQueue: DispatchQueue,
-        callbackQueue: DispatchQueue,
-        completion: @escaping (Result<Manifest, Error>) -> Void
-    ) {
-        do {
-            // find the manifest path and parse it's tools-version
-            let manifestPath = try ManifestLoader.findManifest(packagePath: packagePath, fileSystem: fileSystem, currentToolsVersion: currentToolsVersion)
-            let manifestToolsVersion = try ToolsVersionParser.parse(manifestPath: manifestPath, fileSystem: fileSystem)
-            // validate the manifest tools-version against the toolchain tools-version
-            try manifestToolsVersion.validateToolsVersion(currentToolsVersion, packageIdentity: packageIdentity, packageVersion: packageVersion?.version?.description ?? packageVersion?.revision)
+        delegateQueue: DispatchQueue
+    ) async throws -> Manifest {
+        // find the manifest path and parse it's tools-version
+        let manifestPath = try ManifestLoader.findManifest(packagePath: packagePath, fileSystem: fileSystem, currentToolsVersion: currentToolsVersion)
+        let manifestToolsVersion = try ToolsVersionParser.parse(manifestPath: manifestPath, fileSystem: fileSystem)
+        // validate the manifest tools-version against the toolchain tools-version
+        try manifestToolsVersion.validateToolsVersion(currentToolsVersion, packageIdentity: packageIdentity, packageVersion: packageVersion?.version?.description ?? packageVersion?.revision)
```

Comment:
formatting and accessibility nit: long lines are not readable when not viewed on ultra-wide screens.
```suggestion
        try manifestToolsVersion.validateToolsVersion(
            currentToolsVersion,
            packageIdentity: packageIdentity,
            packageVersion: packageVersion?.version?.description ?? packageVersion?.revision
        )
```